ui/src/Credentials.ts

```diff
@@ -58,19 +60,23 @@ export function retrieveCredentials(): Credentials | undefined {
 
 export function computeToken(party: string): string {
   const payload = {
+    exp: new Date().getTime() / 1000 + 1000000,
     'https://daml.com/ledger-api': {
-      ledgerId: ledgerId,
+      ledgerId: 'da-marketplace-sandbox',
       applicationId: APPLICATION_ID,
       actAs: [party],
-      readAs: [party, publicParty],
+      readAs: [party, 'Public'],
     },
+    party,
+    ledgerId: 'da-marketplace-sandbox',
+    partyName: party,
```

Comment:
Just wondering why you're hardcoding these values ? I presume they were earlier just static variables from some config file.

Reply:
In short, simply because this function to compute a token is only ever run locally (dev), and these values will always be the same for dev. 

On top of that, we don't need to know the `ledgerId` anywhere else in the app anymore (on Hub, the JSON API is now relative to the base domain), and the `usePublicParty()` hook is used to dynamically fetch the correct public ID in whatever running environment the app is in

Reply:
Thanks!